03-attribute-operations.Rmd

```diff
@@ -780,9 +780,10 @@ Calculate the change in percentages and map them.
 1. Using `us_states` and `us_states_df` create a new object called `us_states_sel`.
 The new object should have only two variables - `median_income_15` and `geometry`.
 Change the name of the `median_income_15` column to `Income`.
-1. Calculate the change in median income between 2010 and 2015 for each state.
-Bonus: What was the minimum, average and maximum median income in 2015 for each region?
-What is the region with the largest increase of the median income?
+1. Calculate the change in the number of residents living below the poverty level between 2010 and 2015 for each state. (Hint: See ?us_states_df for documentation on the poverty level columns.)
```

Comment:
Very clear question. Easier for beginners than going straight into percentages. Thank you @zmbc !